### ISSUE TITLE
Update to 1.6.1  error

### ISSUE THREAD

**Issue by strangecarr** [3DJakob/react-tinder-card#120]
Hi, I just updated to 1.6.1 from 1.5.4 and have a module error - 
any one else had similar issue? Here is the trace:

````
Module not found: Package path ./unstable-native-dependencies is not exported from package /xxxx/node_modules/react-dom (see exports field in /xxxx/package.json)

Import trace for requested module:
./node_modules/react-tinder-card/index.js

**Comment by shubhurwt2001** [comment]
Have you found the solution ?

**Comment by strangecarr** [comment]
Unfortunately not, I went back to version 1.5.4 and all works fine again. Are you getting the same error?

**Comment by shubhurwt2001** [comment]
Yes, same error

**Comment by GlaucoNeves** [comment]
Same

**Comment by shubhurwt2001** [comment]
Can anyone help with issue #121

**Comment by 3DJakob** [comment]
@strangecarr @shubhurwt2001 @GlaucoNeves  This has been fixed in PR https://github.com/3DJakob/react-tinder-card/pull/124 and released in v1.6.2 🎉

Please run `npm install react-tinder-card@1.6.2` to resolve the issue.
